csharp/ql/src/semmle/code/csharp/dataflow/FlowSummary.qll

```diff
@@ -135,6 +136,15 @@ module SummaryOutput {
     result = TDelegateSummaryOutput(i, j) and
     hasDelegateArgumentPosition2(c, i, j)
   }
+
+  /**
+   * Gets an output specification that specifies the output `output` of calling
+   * `target` as the output. `output` is limited to (this) parameters and
+   * ordinary returns.
+   */
```

Comment:
I think there should be a more thorough explanation of `JumpSummaries` somewhere. (Either here or at `SummaryJumpNode`). I couldn't really parse/understand this comment. My understanding is that there can be flow summaries where the flow into a callable might impact the flow out of other callables. And therefore we need a way to specify the "other" callable to be able to specify the output path.